.github/actions/tests/checkout-tyk-pro/action.yaml

```diff
@@ -0,0 +1,17 @@
+name: 'Checkout test code'
+description: 'On workflow job failure, fetch and send relevant logs with branch information to an external API'
+author: 'konrad'
+inputs:
+  org_gh_token:
```

Comment:
This is already an org level secret. No need to pass it and risk disclosure in logs, etc.

Reply:
I think that in reusable action we can't use secrets. I've tried again and still have the same issue: https://github.com/TykTechnologies/tyk-pump/actions/runs/15210967071/job/42784948278#step:1:28
when using it set up this way:
```
runs:
  using: "composite"
  steps:
    - name: fetch env from tyk-pro
      shell: bash
      env:
        GH_TOKEN: ${{ secrets.ORG_GH_TOKEN }}
      run: |
        gh release download --repo github.com/TykTechnologies/tyk-pro --archive tar.gz -O env.tgz
        mkdir auto && tar --strip-components=1 -C auto -xzvf env.tgz
```
I've read about it and secrets needs to be passed as inputs in reusable actions

Reply:
Secrets works in reusable workflows, in reusable action there is no syntax to work with secrets.
Also, this api secret can be only used in on-demand created environment.